solutions/izho-14-bank.mdx

```diff
@@ -2,34 +2,185 @@
 id: izho-14-bank
 source: IZhO 2014
 title: Bank
-author: Benjamin Qi
+author: Kevin Sheng
 ---
 
+We do dynamic programming on all subsets of the notes.
+
+Let $\text{covered}[S]$ be the maximum prefix of people we can pay off using the notes defined in the subset, and $\text{leftover}[S]$ be the leftover amount of money we have after we pay off the prefix of the people.
```

Comment:
texttt

Reply:
what's the difference

Reply:
looks better

> Use `\texttt{}` around variable names with length *greater than one*, like so. Place long equations on separate lines with display math, and use `\cdot` instead of `*` to denote multiplication.

*https://usaco.guide/general/adding-solution?lang=cpp#steps*